changelog.d/11441.misc

```diff
@@ -0,0 +1 @@
+Add a test case to sanity check that we parse `/send_join` responses correctly.
```

Comment:
Can we group it with the changelog from https://github.com/matrix-org/synapse/pull/11439?